Formula/g/go-parquet-tools.rb

```diff
@@ -21,15 +22,15 @@ def install
       -s -w
       -X github.com/hangxie/parquet-tools/cmd.version=v#{version}
       -X github.com/hangxie/parquet-tools/cmd.build=#{time.iso8601}
-      -X github.com/hangxie/parquet-tools/cmd.gitHash=#{tap.user}
-      -X github.com/hangxie/parquet-tools/cmd.source=source
+      -X github.com/hangxie/parquet-tools/cmd.gitHash=#{Utils.git_short_head}
+      -X github.com/hangxie/parquet-tools/cmd.source=Homebrew
```

Comment:
If your git tag is immutable, then sha and tag are the same thing, no need to track both tag and sha in the brew build

Reply:
Yes tag is meant to be immutable, deprecate for now and I will remove it from next release.

Reply:
yeah, commitHash good for dirty build, not good for release build 👍